### ISSUE TITLE
e2e flake: Horizontal pod autoscaling (scale resource: CPU) [Feature:Autoscaling] ReplicationController Should scale from 1 pod to 3 pods and from 3 to 5

### ISSUE THREAD

**Issue by piosz** [kubernetes/kubernetes#20327]
```
Horizontal pod autoscaling (scale resource: CPU) [Feature:Autoscaling] ReplicationController Should scale from 1 pod to 3 pods and from 3 to 5

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/horizontal_pod_autoscaling.go:55 Jan 29 03:15:16.236: Number of replicas has changed: expected 3, got 2


Horizontal pod autoscaling (scale resource: CPU) [Feature:Autoscaling] Deployment Should scale from 5 pods to 3 pods and from 3 to 1

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/horizontal_pod_autoscaling.go:48 Jan 29 03:23:13.382: Number of replicas has changed: expected 3, got 2
```

related to #20323

http://kubekins.dls.corp.google.com/view/Critical%20Builds/job/kubernetes-e2e-gce-autoscaling/2631/

**Comment by piosz** [comment]
cc @fgrzadkowski @mwielgus

**Comment by mwielgus** [comment]
Yep, it seems to be some flaky behavior there. Last run failed only 1 test:

Summarizing 1 Failure:

[Fail] Horizontal pod autoscaling (scale resource: CPU) [Feature:Autoscaling] ReplicationController [It] Should scale from 1 pod to 3 pods and from 3 to 5 
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/autoscaling_utils.go:256

Ran 4 of 217 Specs in 4140.259 seconds
FAIL! -- 3 Passed | 1 Failed | 0 Pending | 213 Skipped --- FAIL: TestE2E (4140.54s)
FAIL

Ginkgo ran 1 suite in 1h9m1.010564081s
Test Suite Failed

**Comment by piosz** [comment]
fixed by #20329
